src/dotnet/APIView/APIView/Model/V2/ReviewLine.cs

```diff
@@ -0,0 +1,148 @@
+         *  Similarly all method level code lines are added as children of it's class code line.*/
+        public List<ReviewLine> Children { get; set; } = [];
+        /*** This is set if API is marked as hidden */
+        public bool? IsHidden { get; set; }
+
+        // Following properties are helper methods that's used to render review lines to UI required format.
+        [JsonIgnore]
+        public DiffKind DiffKind { get; set; } = DiffKind.NoneDiff;
+        [JsonIgnore]
+        public bool IsActiveRevisionLine = true;
+        [JsonIgnore]
+        public bool IsDocumentation => Tokens.Count > 0 && Tokens[0].IsDocumentation == true;
+        [JsonIgnore]
+        public bool IsEmpty => Tokens.Count == 0 || !Tokens.Any( t => t.SkipDiff != true);
+        [JsonIgnore]
+        public bool Processed { get; set; } = false;
+
+        public void Add(ReviewToken token)
+        {
+            Tokens.Add(token);
+        }
+
+        public void RemoveSuffixSpace()
+        {
+            if (Tokens.Count > 0)
+            {
+                Tokens[Tokens.Count - 1].HasSuffixSpace = false;
+            }
+        }
+
+        public void AddSuffixSpace()
+        {
+            if (Tokens.Count > 0)
+            {
+                Tokens[Tokens.Count - 1].HasSuffixSpace = true;
+            }
+        }
+
+        public void GetApiText(StringBuilder sb, int indent = 0, bool skipDocs = true)
+        {
+            if (skipDocs && Tokens.Count > 0 && Tokens[0].IsDocumentation == true)
+            {
+                return;
+            }
+
+            //Add empty line in case of review line without tokens
+            if (Tokens.Count == 0)
+            {
+                sb.Append(Environment.NewLine);
+                return;
+            }
+            //Add spaces for indentation
+            for (int i = 0; i < indent; i++)
+            {
+                sb.Append("    ");
+            }
+            //Process all tokens
+            sb.Append(ToString(true));
+            
+            sb.Append(Environment.NewLine);
+            foreach (var child in Children)
+            {
+                child.GetApiText(sb, indent + 1, skipDocs);
+            }
+        }
+
+        private string ToString(bool includeAllTokens)
+        {
+            var filterdTokens = Tokens.Where(x => includeAllTokens || x.SkipDiff != true);
+            if (!filterdTokens.Any())
+            {
+                return "";
+            }
+            StringBuilder sb = new();
+            foreach (var token in filterdTokens)
+            {
+                sb.Append(token.Value);
+                sb.Append(token.HasSuffixSpace == true ? " " : "");
+            }
+            return sb.ToString();
+        }
+
+        
+        public override string ToString()
+        {
+            return ToString(false);
+        }
+
+        public override bool Equals(object obj)
```

Comment:
This seems like an expensive operation - is it used often?

Reply:
This is unused currently. I have added it just in case to use in the future.